From: A. Maitland Bottoms <bottoms@debian.org>
Subject: cmake2.8

Make use of the current version of cmake that comes with
CMakeParseArguments and GNUInstallDirs modules.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -21,7 +21,8 @@
 ########################################################################
 # Project setup
 ########################################################################
-cmake_minimum_required(VERSION 2.6)
+cmake_minimum_required(VERSION 2.8)
+include(GNUInstallDirs)
 project(gr-iqbalance CXX C)
 enable_testing()
 
--- a/cmake/Modules/GrPython.cmake
+++ b/cmake/Modules/GrPython.cmake
@@ -129,7 +129,7 @@
 # Install python sources (also builds and installs byte-compiled python)
 ########################################################################
 function(GR_PYTHON_INSTALL)
-    include(CMakeParseArgumentsCopy)
+    include(CMakeParseArguments)
     CMAKE_PARSE_ARGUMENTS(GR_PYTHON_INSTALL "" "DESTINATION;COMPONENT" "FILES;PROGRAMS" ${ARGN})
 
     ####################################################################
--- a/cmake/Modules/GrSwig.cmake
+++ b/cmake/Modules/GrSwig.cmake
@@ -185,7 +185,7 @@
 ########################################################################
 macro(GR_SWIG_INSTALL)
 
-    include(CMakeParseArgumentsCopy)
+    include(CMakeParseArguments)
     CMAKE_PARSE_ARGUMENTS(GR_SWIG_INSTALL "" "DESTINATION;COMPONENT" "TARGETS" ${ARGN})
 
     foreach(name ${GR_SWIG_INSTALL_TARGETS})
--- a/cmake/Modules/GrMiscUtils.cmake
+++ b/cmake/Modules/GrMiscUtils.cmake
@@ -110,7 +110,7 @@
     endif()
 
     if(GENERATE_LIBTOOL)
-        include(CMakeParseArgumentsCopy)
+        include(CMakeParseArguments)
         CMAKE_PARSE_ARGUMENTS(GR_LIBTOOL "" "TARGET;DESTINATION" "" ${ARGN})
 
         find_program(LIBTOOL libtool)
@@ -130,7 +130,7 @@
 ########################################################################
 function(GR_LIBRARY_FOO target)
     #parse the arguments for component names
-    include(CMakeParseArgumentsCopy)
+    include(CMakeParseArguments)
     CMAKE_PARSE_ARGUMENTS(GR_LIBRARY "" "RUNTIME_COMPONENT;DEVEL_COMPONENT" "" ${ARGN})
 
     #set additional target properties
